build_tools/mako/configuration.py

```diff
@@ -176,7 +176,9 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
               "--iree-vulkan-target-triple=valhall-g77-unknown-android10",
               "--iree-flow-dispatch-linalg-on-tensors",
               "--iree-codegen-spirv-experimental-linalg-on-tensors",
-              "-iree-flow-inline-constants-max-byte-length=2048"
+              # TODO(GH-5330): Replace 16 with 2048 or delete the flag.
```

Comment:
Not sure 2048 is a good configuration. Wit it we will be putting too many constant values into the kernel. 2048 is a workaround for the descriptor issue. I'd just say "revisit the number or delete the flag" here.